### PR TITLE
Change from GFS_ANL to RAP_ANL for surface comparison

### PR DESCRIPTION
**Pull request by XiaSun-Atmos**
**This PR does the following:**

- change GFS_ANL to RAP_ANL for comparison with the GFS model, including 2-m T, 2m RH, gust, composite reflectivity

### CODE REVIEW

File: docs/source/2019HaloweenStorm.rst [hunk 1]
```diff
@@ -110,40 +110,39 @@ Surface Temperature and Wind Speed
...
 
       2-m temperature (F) valid at 00z 1 Nov 2019 
 
-    * Colder 2-m T in MRW_GFSv15p2 along the U.S. east coast compared with GFS_ANL
-    * Colder 2-m T at New England and warmer 2-m T at the Southeast in MRW_GFSv16beta  
+    * Colder 2-m T in MRW_GFSv15p2 along the U.S. east coast compared with RAP_ANL.
+    * Colder 2-m T at New England and warmer 2-m T at the Southeast in MRW_GFSv16beta.
 
-    .. figure:: images/2019Halloween/GUST_MRW_v1.0_2019HalloweenStorm_trim.png
+    .. figure:: images/2019Halloween/GUST_MRW_v1.0_2019HalloweenStorm_RAP_trim.png
```
Comment (climbfuji) [inline docs/source/2019HaloweenStorm.rst:122]: Why is RAP after HalloweenStorm here, and before above in line 113?

File: docs/source/2019HaloweenStorm.rst [hunk 2]
```diff
@@ -110,40 +110,39 @@ Surface Temperature and Wind Speed
...
       :width: 1200
       :align: center
 
       2-m temperature (F) valid at 00z 1 Nov 2019 
 
-    * Colder 2-m T in GFS.v16.0.10 along the U.S. east coast compared with GFS_ANL
+    * Colder 2-m T in GFS.v16.0.10 along the U.S. east coast compared with RAP_ANL.
 
-    .. figure:: images/2019Halloween/GUST_GFS.v16.0.10_2019HalloweenStorm_trim.png
+    .. figure:: images/2019Halloween/GUST_GFS.v16.0.10_2019HalloweenStorm_RAP_trim.png
```
Comment (climbfuji) [inline docs/source/2019HaloweenStorm.rst:139]: Ditto

File: docs/source/2020ColdBlast.rst [hunk 3]
```diff
@@ -124,21 +124,21 @@ Surface Temperature and Wind Speed
 
   .. group-tab:: GFS.v16.0.10
 
-    .. figure:: images/2020JanCold/2mT_GFS.v16.0.10_2020JanCold_trim.png
+    .. figure:: images/2020JanCold/2mT_GFS.v16.0.10_RAP_2020JanCold_trim.png
```
Comment (climbfuji) [inline docs/source/2020ColdBlast.rst]: Ditto

File: docs/source/2020ColdBlast.rst [hunk 4]
```diff
@@ -124,21 +124,21 @@ Surface Temperature and Wind Speed
...
 
-    .. figure:: images/2020JanCold/GUST_GFS.v16.0.10_2020JanCold_trim.png
+    .. figure:: images/2020JanCold/GUST_GFS.v16.0.10_2020JanCold_RAP_trim.png
       :width: 1200
       :align: center
 
       Surface gust (m/s)
 
-    * GFS.v16.0.10 simulates weaker wind speed across Midwest and Ontario.
+    * GFS.v16.0.10 generates stronger winds at Deep North region and weaker winds at Ontario.
```
Comment (climbfuji) [inline docs/source/2020ColdBlast.rst]: over Ontario?

**Review by climbfuji — CHANGES_REQUESTED** [review]
It would be nice if the `_RAP` label in the filename was placed consistenly before or after the case. Right now some files have it before, some after.

File: docs/source/2019HaloweenStorm.rst (hunk 1 above)
Comment (XiaSun-Atmos) [reply docs/source/2019HaloweenStorm.rst:122]: Good catch Dom! Comments are addressed. Thanks!

**Review by climbfuji — APPROVED** [review]

**Pull request merged**